src/__tests__/client.test.ts

```diff
@@ -0,0 +1,22 @@
+import Client from '../client';
+import { PollingProcessor } from '../polling';
+
+jest.mock('../openapi/api');
+
+it('check close client', async () => {
+  // given
+  const start = jest.spyOn(PollingProcessor.prototype, 'start');
+  const close = jest.spyOn(PollingProcessor.prototype, 'close');
+
+  // when
+  const client = new Client('some key', {
+    enableAnalytics: false,
+  });
+  await client.waitForInitialization();
+
+  client.close();
```

Comment:
I haven't checked it locally, but this like might trigger a warning about an action being taken outside of an `act` block. If not, don't worry about it 👍🏻